src/main/java/uk/gov/hmcts/ccd/domain/service/callbacks/retrycontext/CallbackRetryContextBuilder.java

```diff
@@ -0,0 +1,87 @@
+package uk.gov.hmcts.ccd.domain.service.callbacks.retrycontext;
+
+import com.google.common.collect.Iterables;
+import com.google.common.collect.Lists;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.ccd.ApplicationParams;
+
+import java.util.List;
+
+import static java.util.Collections.unmodifiableList;
+
+/**
+ * This builder builds retries context based on the spreadsheet config for retries timeouts. The intervals are fixed and are multiples of 3 starting from
+ * 0s and 1 i.e. 0s, 1s, 3s, 9s, 27s ...
+ *
+ * Blank value means default retries (0s interval with 60s default timeout, 1s interval with 60s default timeout, 3s interval with 60s default timeout).
+ * Single 0 value means disable retries (single send at 0s interval with 60s default timeout).
+ * Any comma separated chain of values that DOES NOT include 0 means custom retries.
+ * Any comma separated chain of values that DOES include 0 means default retries (see blank value).
+ *
+ * The examples input callback retry timeouts and output callback retry contexts are as follows:
+ *
+ *      blank - default retries context (0s interval with 60s default timeout, 1s interval with 60s default timeout, 3s interval with 60s default timeout)
+ *      0 - disable retries (0s interval with 60s default callbacks timeout)
+ *      1 - custom retries (0s interval with 1s custom timeout)
```

Comment:
`1` has no retries isn't it? `custom retries` is a bit misleading

Reply:
`0` is no retries. `1` is part of custom retries.

Reply:
ok but with `1` there are no retries am I correct?

Reply:
Yes with `1` there will be no retries. It will be initial interval of `0` and then a timeout of `1` and that is it.